pkg/apis/provisioning/v1alpha5/kubelet_configuration.go

```diff
@@ -14,12 +14,62 @@ limitations under the License.
 
 package v1alpha5
 
+import (
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/ptr"
+)
+
 // KubeletConfiguration defines args to be used when configuring kubelet on provisioned nodes.
 // They are a subset of the upstream types, recognizing not all options may be supported.
-// Wherever possible, the types and names should reflect the upstream kubelet types.
+// Wherever possible, the types and names should reflect the upstream kubelet types from
```

Comment:
>  Wherever possible, 

This concerns me. This spec should be an explicit, word for word copy paste of snippits from that struct.